### PR TITLE
version/doc changes

### PR DESCRIPTION
**Pull request by xiangyan99**

### CODE REVIEW

**Comment by adxsdk6** [comment]
Can one of the admins verify this patch?

File: sdk/appconfiguration/azure-appconfiguration/README.md [hunk 1]
```diff
@@ -46,15 +50,74 @@ az appconfig credential list --name <config-store-name>
...
+Here we demonstrate using [DefaultAzureCredential][default_cred_ref]
+to authenticate as a service principal. However, [AzureAppConfigurationClient][configuration_client_class]
+accepts any [azure-identity][azure_identity] credential. See the
+[azure-identity][azure_identity] documentation for more information about other
+credentials.
+
+##### Create a service principal (optional)
+This [Azure CLI][azure_cli] snippet shows how to create a
+new service principal. Before using it, replace "your-application-name" with
+a more appropriate name for your service principal.
```
Comment (bryevdv) [inline sdk/appconfiguration/azure-appconfiguration/README.md]: suggest: "with the appropriate name..."

File: sdk/appconfiguration/azure-appconfiguration/README.md [hunk 2]
```diff
@@ -46,15 +50,74 @@ az appconfig credential list --name <config-store-name>
...
+
+##### Create a client
+Once the **AZURE_CLIENT_ID**, **AZURE_CLIENT_SECRET** and
+**AZURE_TENANT_ID** environment variables are set,
+[DefaultAzureCredential][default_cred_ref] will be able to authenticate the
+[AzureAppConfigurationClient][configuration_client_class].
+
+Constructing the client also requires your configuration store's URL, which you can
+get from the Azure CLI or the Azure Portal. In the Azure Portal, this URL is
+the service's "Endpoint".
```
Comment (bryevdv) [inline sdk/appconfiguration/azure-appconfiguration/README.md]: Should Endpoint be capitalized?

Comment (bryevdv) [reply sdk/appconfiguration/azure-appconfiguration/README.md]: I guess maybe, but perhaps: "In the Azure Portal, the URL can be found listed as the service "Endpoint".

**Comment by bryevdv** [comment]
Some tiny suggestions otherwise LGTM

File: sdk/appconfiguration/azure-appconfiguration/README.md (hunk 2 above)
Comment (xiangyan99) [reply sdk/appconfiguration/azure-appconfiguration/README.md]: This is what we show on the portal. But anyway, I will change to lower case.

**Pull request merged**